chromium_src/chrome/browser/component_updater/registration.cc

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../chrome/browser/component_updater/registration.h"
+
+#define RegisterComponentsForUpdate RegisterComponentsForUpdate_ChromiumImpl
+#include "../../../../../chrome/browser/component_updater/registration.cc"  // NOLINT
+#undef RegisterComponentsForUpdate
+
+#include "brave/components/brave_wallet/browser/wallet_data_files_installer.h"
```

Comment:
This header should be guarded with wallet buildflag, same as the usage below in the override.

Reply:
Fixed in 00a17f6f17500a72e33cd0760ab2e297b9db06ec

Reply:
Fixed in 03cfad13253c57cdb043529378d80f08129f2c39